frontend/apps/marketing/README.md

```diff
@@ -2,10 +2,38 @@
 
 ## Getting Started
 
+### Secrets
+
+Contentful API Keys and other secrets will need to be populated for this application to run correctly.
+
+1. Copy `.env.example` to `.env`
+   ```bash
+   cp .env.example .env
+   ```
+2. For each entry in `.env`, see the instructions for how to obtain an API Key.
+
+**Note**: `.env` must not be committed to source. It is git ignored by default.
+
+### Developing
+
 First, run the development server:
 
 ```bash
 yarn dev
 ```
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```

Comment:
Should we also include some info about how to use this on the Contentful side (eg, setting the appropriate "Preview Platform" like shown in [this thread?](https://codedotorg.slack.com/archives/C07UW4ED66Q/p1733441644442949?thread_ts=1733440885.431029&cid=C07UW4ED66Q))

Also, longer term, but would we eventually include some guidance to use the `dev` Contentful environment?

Reply:
I'm thinking once we have a good rhythm we can go back and create the correct docs. Right now things are in flight with the temporary AWS setup and I'm afraid we'll leave some outdated information. I have a task to slot that in perhaps in mid to late January to back fill all documentation since I'm guessing we'll be getting in the rhythm of creating and integrating by then.